documentation/pom.xml

```diff
@@ -0,0 +1,307 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>validation-api-spec</artifactId>
+        <groupId>org.beanvalidation</groupId>
+        <version>2.0.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>documentation</artifactId>
+
+    <properties>
+        <bv.version.qualifier></bv.version.qualifier><!-- e.g. ' (Early Draft 1)' -->
+        <maven.build.timestamp.format>yyyy-MM-dd</maven.build.timestamp.format>
+        <bv.revdate>${maven.build.timestamp}</bv.revdate> <!-- Replace with fixed date when releasing -->
+        <license>license-evaluation</license>
+        <pdf.name>bean-validation-specification.pdf</pdf.name>
+
+        <examples-source-dir.path>${project.build.directory}/examples/</examples-source-dir.path>
+        <validation-api-source-dir.path>${project.build.directory}/validation-api-sources/</validation-api-source-dir.path>
+
+        <asciidoctor-maven-plugin.backend>html5</asciidoctor-maven-plugin.backend>
+        <asciidoctor-maven-plugin.sourceHighlighter>coderay</asciidoctor-maven-plugin.sourceHighlighter>
+        <asciidoctor-maven-plugin.outputDirectory>${project.build.directory}/${asciidoctor-maven-plugin.backend}</asciidoctor-maven-plugin.outputDirectory>
+    </properties>
+
+    <dependencies>
+        <dependency>
```

Comment:
good question :)
I've took it from this example:
https://github.com/asciidoctor/asciidoctor-maven-examples/blob/master/asciidoctor-pdf-with-theme-example/pom.xml